# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/nikolaus/unison/unison-Linux.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN unison-2.9.20/ubase/util.ml unison-2.9.20-avm/ubase/util.ml
--- unison-2.9.20/ubase/util.ml	2002-08-19 23:15:04.000000000 +0200
+++ unison-2.9.20-avm/ubase/util.ml	2003-11-10 22:46:21.000000000 +0100
@@ -217,7 +217,7 @@
     close_in c;
     match l with
       "Darwin" -> true
-    | "linux" | "FreeBSD" | "SunOS" -> false
+    | "Linux" | "FreeBSD" | "SunOS" -> false
     | _ ->
         Printf.printf
          "Warning: 'uname' returned unrecognized result (%s) in Util.isOSX.\n"
